include/PrimeTower.h

```diff
@@ -1,15 +1,16 @@
-//Copyright (c) 2022 Ultimaker B.V.
-//CuraEngine is released under the terms of the AGPLv3 or higher.
+// Copyright (c) 2022 Ultimaker B.V.
+// CuraEngine is released under the terms of the AGPLv3 or higher.
 
 #ifndef PRIME_TOWER_H
 #define PRIME_TOWER_H
 
-#include <vector>
-
+#include "settings/types/LayerIndex.h"
 #include "utils/polygon.h" // Polygons
 #include "utils/polygonUtils.h"
 
-namespace cura 
+#include <vector>
```

Comment:
Style:  C++ standard library headers, other libraries' headers, your project's headers.

Reply:
You're right ! I thought the clang-format was taking care of that, but it was wrong :)